exporter/awsemfexporter/metric_translator.go

```diff
@@ -421,3 +468,221 @@ func needsCalculateRate(pmd *pdata.Metric) bool {
 	}
 	return false
 }
+
+// convert map of GroupedMetric objects into map format for compatible with PLE input
+func TranslateBatchedMetricToEMF(groupedMetricMap map[string]*GroupedMetric) []*LogEvent {
+	ples := make([]*LogEvent, 0, maximumLogEventsPerPut)
+	for _, v := range groupedMetricMap {
+		fieldMap := make(map[string]interface{})
+		labelsList := make([][]string, 0)
+		metricsList := make([]map[string]string, 0)
+		fieldMap["Namespace"] = v.Namespace
+
+		lList := make([]string, 0)
+		for key, val := range v.Labels {
+			lList = append(lList,key)
+			fieldMap[key] = val
+		}
+
+		labelsList = append(labelsList, lList)
+
+		for key, val := range v.Metrics {
+			metricDef := make(map[string]string)
+			metricDef["Name"] = key
+			metricDef["Unit"] = val.Unit
+			metricsList = append(metricsList, metricDef)
+			fieldMap[key] = val.Value
+		}
+
+		cwm := &CWMeasurement {
+			Namespace: v.Namespace,
+			Dimensions: labelsList,
+			Metrics: metricsList,
+		}
+
+		cwmMap := make(map[string]interface{})
+		cwmMap["CloudWatchMetrics"] = cwm
+		cwmMap["Timestamp"] = v.Timestamp
+		fieldMap["_aws"] = cwmMap
+
+		pleMsg, err := json.Marshal(fieldMap)
+		if err != nil {
+			continue
+		}
+		metricCreationTime := v.Timestamp
+
+		logEvent := NewLogEvent(
+			metricCreationTime,
+			string(pleMsg),
+		)
+		logEvent.LogGeneratedTime = time.Unix(0, metricCreationTime*int64(time.Millisecond))
+		ples = append(ples, logEvent)
+	}
+	return ples
+}
+
+// TranslateOtToCWMetric converts OT metrics to CloudWatch Metric format
+func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, dimensionRollupOption string, namespace string) ([]*CWMetrics, int) {
```

Comment:
Why is this inserted here? This might be because you removed it but then inserted it back in, causing the git diff to think this is new code. This makes it a bit difficult to review as we can't guarantee that this is the exact same code. Can you fix this?